README.md

```diff
@@ -458,6 +458,41 @@ We use it for the control container because it needs to be available early to gi
 
 Be careful, and make sure you have a similar low-level use case before reaching for host containers.
 
+#### Bootstrap containers settings
+* `settings.bootstrap-containers.<name>.source`: The URI of the container
+* `settings.bootstrap-containers.<name>.mode`: The mode of the container, it could be one of `off`, `once` or `always`
+* `settings.bootstrap-containers.<name>.user-data`: field with arbitrary base64-encoded data
+
+Bootstrap containers are host containers that can be used to "bootstrap" the host before services like ecs-agent, kubelet and docker start.
```

Comment:
nit: "ecs-agent, kubelet, and docker start" to use the Oxford comma.

second nit: those are the names of the services in systemd, which is one correct way to do this.  The other way would be to use the names of the software which would be "ECS Agent", "Kubelet", and "Docker".  Is it your intention to use the systemd service names?

Reply:
No, my intention was to used the services names. I already updated them.